roles/finalize_deploy/tasks/main.yml

```diff
@@ -9,6 +9,30 @@
   become: true
   become_user: "{{ deploy_user }}"
   block:
+    - name: Create "next" symlink for any final steps before switching live
+      file:
+        src: "{{ deploy }}"
+        dest: "{{ install_root }}/next"
+        state: link
+      tags:
+        - never
+        - final-pause
+
+    - name: pause before switching live
+      ansible.builtin.pause:
+        prompt: "Do any steps needed before the new version is live, e.g., reindexing. (CTRL-C when done)"
+      tags:
+        - never
+        - final-pause
```

Comment:
I like using `never` with a second tag - that's a nice way to make the playbook run or not run those tasks in the right scenarios and also have the tags you pass feel more sensible.